frontend/src/components/Main/Main.tsx

```diff
@@ -0,0 +1,74 @@
+import './Main.css';
+import adoptImage from '../../images/adopt-pets.png';
+import donateImage from '../../images/dogbrush.png';
+import sponsorImage from '../../images/cat-mailbox.png';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Main = () => {
+  return (
+    <main className="main">
+      <section className="section section_adopt">
+        <div className="section__items">
+          <h2 className="section__title">Adopt pets in your area</h2>
+          <p className="section__text">
+            Find cats, dogs and other pets available right near you.
+          </p>
+          <Link to="find-pets">
+            <button className="section__button section__button_red">
+              Adopt
```

Comment:
Buttons have excessive padding on the top on my end. Also, the margin on the left makes it unaligned with the text.

Maybe we should downsize both buttons and fonts or use responsive sizes (like em).

line-height property also adds more size to the button. You might wanna remove all classes and properties and start from scratch, adding one property one by one, because there is some style overlap, specially in the .section__button class.

![image](https://user-images.githubusercontent.com/15349795/197787765-bd9b45a1-7c00-4580-b930-1be44306e48b.png)

Reply:
Updated